### PR TITLE
Migrated travis ci to github actions

### PR DESCRIPTION
**Pull request by alexpalchuk**
Migrated travis ci to github actions. There are several thing we need to do:

- [x] Migrate $CODESIGNING variable to be usable by github actions
- [x] Make sure that release to maven works so need to adjust `settings.xml `with creds
- [x] Disable travis ci on repo level

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: build
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```
Comment (barakm) [inline .github/workflows/build.yml:5]: Why an old version of Ubunutu? Is this the most up to date version that GH action supports?

Comment (alexpalchuk) [reply .github/workflows/build.yml:5]: according to https://github.com/actions/virtual-environments
we can update to Ubuntu 20.04

Comment (barakm) [reply .github/workflows/build.yml:5]: Cool, let's do it.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '8'
+          cache: 'maven'
+
+      - name: Install
+        run: |
+          echo 'Install on ' ${GITHUB_REF##*/}
+          ./mvnw install -DskipTests  -Dgpg.skip
```
Comment (barakm) [inline .github/workflows/build.yml:20]: Why not run the unit tests here? Do we run them somewhere else?

File: .github/workflows/build.yml (hunk 1 above)
Comment (alexpalchuk) [reply .github/workflows/build.yml:5]: on it

File: .github/workflows/build.yml (hunk 2 above)
Comment (alexpalchuk) [reply .github/workflows/build.yml:20]: well, I did the same flow as travis has with only install, for example https://github.com/logzio/sawmill/blob/master/.travis.yml#L13

Comment (barakm) [reply .github/workflows/build.yml:20]: That looks like a bug. Someone, somewhere, should be running the unit tests.

Comment (alexpalchuk) [reply .github/workflows/build.yml:20]: I just checked some merged PR and it has only `install` and `deploy` stages https://travis-ci.org/github/logzio/sawmill/builds/753004033
If we want to have also test run I can add this stage if no objections.

Comment (barakm) [reply .github/workflows/build.yml:20]: Let's finish migrating to GH actions as is. Once this works, let's enable tests as well.

**Review by barakm — APPROVED** [review]

**Comment by alexpalchuk** [comment]
@barakm updated

**Review by barakm — APPROVED** [review]

**Pull request merged**